src/routes/webhooks/polar.ts

```
@@ -0,0 +1,185 @@
+        subscriptionModified = true;
+        break;
+
+      case 'subscription.cancelled':
+        userSubscription.status = 'cancelled';
+        // Bạn có thể chọn hết hạn tín dụng ngay lập tức hoặc sau một thời gian ân hạn.
+        // Trong ví dụ này, tín dụng vẫn còn nhưng trạng thái bị hủy, ngăn chặn việc sử dụng thêm.
+        subscriptionModified = true;
+        break;
+
+      case 'subscription.tier_changed':
+        const newTier = polarEvent.payload.new_product_id;
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)